run_iwyu_tests.py

```diff
@@ -99,17 +99,77 @@ def setUp(self):
       'lambda_fwd_decl.cc': ['-std=c++11'],
       'lateparsed_template.cc': ['-fdelayed-template-parsing'],
       'ms_inline_asm.cc': ['-fms-extensions'],
-      'prefix_header_attribution.cc': [
-          self.Include('prefix_header_attribution-d1.h')],
+      'prefix_header_attribution.cc': [self.Include('prefix_header_attribution-d1.h')],
       'prefix_header_includes_add.cc': prefix_headers,
       'prefix_header_includes_keep.cc': prefix_headers,
       'prefix_header_includes_remove.cc': prefix_headers,
     }
+    include_map = {
```

Comment:
So will we be able to shrink this as your actual IWYU patches come in? (hoping!)